src/main/java/frc/robot/HardwareMap.java

```diff
@@ -12,20 +12,47 @@ public final class HardwareMap {
 	public static final int CAN_ID_SPARK_DRIVE_BACK_RIGHT = 2;
 	public static final int CAN_ID_SPARK_DRIVE_FRONT_LEFT = 3;
 	public static final int CAN_ID_SPARK_DRIVE_BACK_LEFT = 4;
-	public static final int CAN_ID_SPARK_SHOOTER = 5;
+	public static final int CAN_ID_ARM = 10;
+	public static final int CAN_ID_SPINNER_MOTOR = 2;
+	public static final int CAN_ID_FLIP_MOTOR = 5;
+	public static final int CAN_ID_WRIST_MOTOR = 6;
 
 	// Pneumatics channel numbers
 	public static final int PCM_CHANNEL_INTAKE_CYLINDER_FORWARD = 1;
 	public static final int PCM_CHANNEL_INTAKE_CYLINDER_REVERSE = 2;
 
-	// Place jumper from DIO pin 9 to GND to indicate this is a test setup
-	private static final int DIO_TEST_SETUP_CHANNEL = 9;
-	private static DigitalInput testBoardPin = new DigitalInput(HardwareMap.DIO_TEST_SETUP_CHANNEL);
+	// Place jumper from DIO pin 9 to GND to indicate this is a test setu setup
+	private static final int DIO_SPINNING_INTAKE_CHANNEL = 9;
+	private static final int DIO_ELEVATOR_ARM_CHANNEL = 8;
+	private static final int DIO_WRIST_CHANNEL = 7;
+
+	private static DigitalInput testBoardPin = new DigitalInput(
```

Comment:
testBoardPin doesn't line up with DIO_SPINNING_INTAKE_CHANNEL, what exactly is this pin used for?

Reply:
Renamed testBoardPin to spinning intake pin. This pin is for disabling the intake system when testing.